SR2019/2018-09-27-kickstart-announcement.md

```diff
@@ -0,0 +1,16 @@
+---
+to: All Teams
+subject: SR2019 Registration and Kickstart Announcement
+---
+
+After a year's hiatus during which we've worked to ensure that Student Robotics will be more scalable for the future, we're pleased to announce the continuation of the annual competition. As always, we start this competition period with a Kickstart event. Here, we will announce the game, and show you how to use the kit.
+
+In order to allow the core team of volunteers to focus on delivering a high quality programme while requiring far fewer resources than before, this year's competition will be slightly smaller than previous years. We expect to be able to increase the size of the competition again in the future.
+
+Kickstart will be on **Saturday 10th November**. This time, we're planning on running 2 events. 1 at the **University of Southampton**, and the second in **London**. The exact locations will be confirmed in the next couple of weeks, however the events will run from around 10am, and finish some time before 5pm. This information will be reflected on our website soon. If you are unable to make either of these, please let us know!
+
+If you would like to compete in Student Robotics 2019, please fill in the form at the link below with the required information. All further emails about SR2019 will be sent there. The closing date will be *1st November*.
```

Comment:
Maybe mention teams will be picked on a first-come first-served basis?

Reply:
We never decided this, so let's leave it until we're sure that's how we're going to do it. I'll add something about responding the sooner the better